### DISCUSSION TITLE
Onedrive挂载添加成功但无法加载文件列表

### DISCUSSION THREAD

**Discussion by edge-sky** [AlistGo/alist#7334] [Q&A] [answered]
挂载配置如下
![1e3c850500fbf4cbc6b9a8ac696d81e6](https://github.com/user-attachments/assets/f801e555-b329-45a1-b7c1-bc8c99d7374c)

在主页中打开文件列表出现错误如下
![image](https://github.com/user-attachments/assets/d85ab69e-6649-43f9-be2e-b0cf7c2d9c63)

我使用的挂载教程为 https://alist.nn.ci/zh/guide/drivers/onedrive.html
显示添加成功，权限也已开启，但依旧出现该错误，搜索后并未找到相关的解决方案

**Comment by Simonzhang8** [comment, marked as answer]
根文件夹路径指的是OneDrive内的路径 不是服务器的路径
那个位置填/ 或者根据你实际需要挂载的OneDrive内路径填写

**Comment by edge-sky** [comment]
感谢！我前面在搞本地挂载，惯性思维带入了，已经解决


---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2024年10月14日(周一) 下午5:40
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [alist-org/alist] Onedrive挂载添加成功但无法加载文件列表 (Discussion #7334)




 
根文件夹路径指的是OneDrive内的路径 不是服务器的路径
 那个位置填/ 或者根据你实际需要挂载的OneDrive内路径填写
 
&mdash;
